openapi/cloudmersive.barcode/client.bal

```diff
@@ -40,89 +40,90 @@ public isolated client class Client {
         http:Client httpEp = check new (serviceUrl, clientConfig);
```

Comment:
The official OpenAPI of Cloudmersive API contains `https://testapi.cloudmersive.com/` as the default URL and it can be used for testing purposes. `https://api.cloudmersive.com` also can be used in free tier. It is the users choice to use the URL accordingly. Any of these URLs will be fine

Reply:
Ack